From 12bde024e868702f0e703990c5ecaf9f8ca674ce Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 11 Mar 2022 17:24:37 +0000
Subject: [PATCH] drm: Include drm_connector.h from drm_panel.h

drm_panel.h wants to reference enum drm_panel_orientation which is defined
in drm_connector.h (despite the name).
Include drm_connector.h in drm_panel.h to avoid the rare situation where
drm_panel.h is used with drm_connector.h

https://github.com/raspberrypi/linux/issues/4919

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 include/drm/drm_panel.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/drm/drm_panel.h b/include/drm/drm_panel.h
index 81ea5e5b272e..451f88efcd1c 100644
--- a/include/drm/drm_panel.h
+++ b/include/drm/drm_panel.h
@@ -24,6 +24,7 @@
 #ifndef __DRM_PANEL_H__
 #define __DRM_PANEL_H__
 
+#include <drm/drm_connector.h>
 #include <linux/err.h>
 #include <linux/errno.h>
 #include <linux/list.h>
@@ -36,8 +37,6 @@ struct drm_device;
 struct drm_panel;
 struct display_timing;
 
-enum drm_panel_orientation;
-
 /**
  * struct drm_panel_funcs - perform operations on a given panel
  *
-- 
2.35.5

